Feat/mixed/hotfixes n additions

Grekopithikos opened the pull request:
<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
- Added lots of characters
- Moved the `wc_race_portrait_modifiers` from `portrait_modifiers` to `trait_portrait_modifiers`, which would hopefully give better performance (Fixes issue #1712 )
- Valormok is ruled by the orc Ag'tor Bloodfist under Durotar in 605
- Stormstout-Min and Stormstout-Mab are now two houses in the same Stormstout dynasty
- Stormstout county is now ruled by house Stormstout-Min
- Renamed Pharamond Rivendare to Titus
- Renamed the Townlong culture to Yaungol
- Adventurer government characters can properly marry
- Replaced the Warmonger tenet in Theredrasism with Ancestor Worship as the Dark Fanaticism doctrine makes it redundant
- Ordosism has water magic enslaved instead of criminal
- Added the Ping culture to represent humans of Asian ethnicity (all 3 of them)
- Readded the Veiled Blade adventurer camp
- Added the Marshal Expeditions adventurer camp
- The Dire trait is no longer assigned to random races that are not trolls (Fixes issue #1723 )
- Made a fix for `Call for Confederation` Decision
- Characters and their vassals who flee from the scourge and travel to Kalimdor will now get the trait disinherited once Theramore is formed
- Made a fix where `Commission Epic` decision was available for everyone even if they didn't have the required perk to unlock the decision
- Made a fix where the Vassalization Casus Belli wasn't available after getting the `forced_vassalage_perk`
- Flowerpicker culture
- Non-player Tauren characters with the nickname "Brave" are more uncommon
- If a non-player Tauren already has a nickname they won't get the nickname "Brave"
- Religions will only get access to Great Holy Wars after the year 615
- Religions can now only target Great Holy Wars for their own Holy Sites (Fixes issue #1661 )
- Balancing Fix: Garithos won't be in the Alliance Union nor can he join it (Fixes issue #1733 )
- Chris Metzen
- Characters who can breathe underwater don't drown in events
- Characters who can't be poisoned don't get poisoned in events
- Characters who can fly don't fall to their deaths
- Patron characters have a modifier that prevents them from autogenerating families
- Every historic adult tortollan has a set appearance that reflects how they look like in WoW
- Added a single snobold character, Kaganishu who is a courtier under Magmothregar
- Administrative and Nomad governments can now join unions
- Fixed an issue where southsea and some other ethnicities characters looked green
- Added lore names for demonic and argusean cultures
- Added the Flame's Radiance renown icon as an icon that can be picked for custom Tyrist faiths, and the K'aresh Trust and Manaforge Vandals icons for the void faiths

<!--
A changelog where you can describe more complicated things to other developers and testers.
-->
## Developer changelog:
- Renamed `d_drustwar` to `d_drustvar`.
- Renamed `ear_lenght` to `ear_length`.
- Renamed `shamanism` to `wildhammer_shamanism`.

<!--
Before the merge, we recommend you to do these tests with your branch.
-->
## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

<!--
If you need to explain something to testers. Otherwise, delete this part.
-->
# How to test:

Grekopithikos committed:
wc_historical_character_triggers

Grekopithikos committed:
renamed heritage triggers

Grekopithikos committed:
League of Arathor

Grekopithikos committed:
no slop pls

Grekopithikos committed:
Defilers

Grekopithikos committed:
Belgrom and Keldor

Grekopithikos committed:
Hammerfall NPCs

Grekopithikos committed:
Marshal family

Grekopithikos committed:
Way of the nomad centaurs

Grekopithikos committed:
Way of the Nomad yaungol

Grekopithikos committed:
Update 00_government_types.txt

Grekopithikos committed:
Update 00_wc_landed_titles.txt

Grekopithikos committed:
static portraits moved

FelModder committed:
adding gov limit to call for confederation decision

FelModder committed:
you get disinherited and remove undead trait from vassals if they get those

Grekopithikos committed:
Marshal Expeditions members

Grekopithikos committed:
generated amani troll id change

Grekopithikos committed:
Desolate council members

Grekopithikos committed:
Bloodfist and Adams families

Grekopithikos committed:
lotsa changes

Grekopithikos committed:
Yaungol characters

Grekopithikos committed:
lots of characters

FelModder committed:
Dire trait getting assigned to random races fix

FelModder committed:
Revert "no slop pls"

This reverts commit b4b00ec9921beb47bbf0810c287deeaf8adf978c.

FelModder committed:
don't remove those unless you have other handmade stuff to replace it

FelModder committed:
Commission Epic Decision fix

FelModder committed:
Vassalization Casus Belli fix for forced_vassalage_perk

Grekopithikos committed:
some more characters

Grekopithikos committed:
Dreadscorn

Grekopithikos committed:
no more invisible murlocs

also kazzak should be orange

Grekopithikos committed:
readded veiled blade

Grekopithikos committed:
Update 52000_ogre.txt

Grekopithikos committed:
Merge branch 'dev' into feat/mixed/hotfixes-n-additions

Grekopithikos committed:
pandaren update

Grekopithikos committed:
Pearlfin, Inkgill and Fallsong tribes

Grekopithikos committed:
Razorfin tribe

Grekopithikos committed:
more jinyu

Grekopithikos committed:
mudclaw family

Grekopithikos committed:
Stormstout houses

Also the Stormstouts rule the Stormstout county

Grekopithikos committed:
Ping culture

For all 3 East Asian humans

Grekopithikos committed:
Update 00_k_kun_lai.txt

Grekopithikos committed:
Update 00_k_kun_lai.txt

Grekopithikos committed:
more chars

Grekopithikos committed:
Flowerpicker culture

Grekopithikos committed:
Yaungol can do water magic

UDie2day committed:
Corrected "caster" for impaler and turtle rider MaA

UDie2day committed:
corrected another "casters" -> "caster"

Grekopithikos committed:
Flame's Radiance faith icon

UDie2day committed:
impaler type fix, game concept horde alias fix

Grekopithikos committed:
coas for some cultures

FelModder committed:
More 'caster' fix

FelModder committed:
Error log fix

Grekopithikos committed:
Dragon CoA colors

Grekopithikos committed:
Tauren Braves more uncommon

FelModder committed:
Fixing suggestion 294

FelModder committed:
Can only direct GHW towards your own holy sites

FelModder committed:
Also the case with Directed Great Holy Wars

Grekopithikos committed:
some more characters

Grekopithikos committed:
ebon blade is mord'truar heresy

UDie2day committed:
General fixes

* Duplicate next_building = tribe_02
* Duplicate type_icon = "icon_building_graveyard.dds"
* Duplicate world_innovation_elephants and camels
* Localization fixes
* Other Fixes

UDie2day committed:
General fixes

* Localization fixes
* Duplicate dynasty changes
* Duplicate blocks removed (thermore and horde invader)
* Bookmark fixes

Grekopithikos committed:
Marshal Expeditions

Grekopithikos committed:
Stoneplow families

UDie2day committed:
Uther event fix

Allows the uther event and terenas funeral event to trigger when Arthas kills Terenas.

UDie2day committed:
Longbow competition

Longbowmen aren't in the game which causes an error for the tradition, so I re-added them.
Also fixed the localization for the frontier warrior tradition.

FelModder committed:
Garithos cannot join Unions until lordaeron is taken back

FelModder committed:
Okay actual Garithos fix

FelModder committed:
Update 48000_tirassian.txt

Grekopithikos committed:
Garithos doesn't lead the Alliance

Grekopithikos committed:
Narain fix

Grekopithikos committed:
Mishan Waycrest

Grekopithikos committed:
commander xp

Grekopithikos committed:
Update 500000_pandaren.txt

Grekopithikos committed:
Chris Metzen

Chris Metzen

Grekopithikos committed:
Kal'Thuzad fix

Grekopithikos committed:
Tortollan portraits

Grekopithikos committed:
more tortollan portrais

Grekopithikos committed:
Kul Tiran characters

Grekopithikos committed:
Southsea Freebooters

Grekopithikos committed:
Reginald Windsors band and Morgan's Militia characters

Grekopithikos committed:
Characters who can breathe underwater don't drown

Grekopithikos committed:
poisoning and falling won't happen to those who can't be poisoned or fall

Grekopithikos committed:
starvation from lack of provisions don't apply to those who don't need food

Grekopithikos committed:
ragged nickname

Grekopithikos committed:
Update 57000_amani.txt

Grekopithikos committed:
Brotherhood of the Light

Grekopithikos committed:
Farthing and Baloo families

Grekopithikos committed:
Patron character modifier

Grekopithikos committed:
Nathrezim names

Grekopithikos committed:
Update wc_race_portrait_modifiers.txt

Grekopithikos committed:
Demon names

Grekopithikos committed:
Stratholme residents

Grekopithikos committed:
Leper Gnomes

Grekopithikos committed:
K'aresh icons

Grekopithikos committed:
Mangataur update

Grekopithikos committed:
Karazhan npcs

Grekopithikos committed:
Westfall Story

FelModder commented:
✅: Working as intended
☀️: Working but has a few issues
❌: Not working as intended

- Added lots of characters ✅
- Moved the `wc_race_portrait_modifiers` from `portrait_modifiers` to `trait_portrait_modifiers`, which would hopefully give better performance (Fixes issue #1712 ) ☀️ (_From me running it for 23 years in-game, it definitely is slightly better in performance but not much. An improvement nonetheless, hence no need of further changes_)
- All centaurs and yaungol have the Way of the Nomad trait ✅
- Valormok is ruled by the orc Ag'tor Bloodfist under Durotar in 605 ✅
- Stormstout-Min and Stormstout-Mab are now two houses in the same Stormstout dynasty ✅
- Stormstout county is now ruled by house Stormstout-Min ✅
- Renamed Pharamond Rivendare to Titus ✅
- Renamed the Townlong culture to Yaungol ✅
- Adventurer government characters can properly marry ✅
- Replaced the Warmonger tenet in Theredrasism with Ancestor Worship as the Dark Fanaticism doctrine makes it redundant ✅
- Ordosism has water magic enslaved instead of criminal ✅
- Added the Ping culture to represent humans of Asian ethnicity (all 3 of them) ✅
- Readded the Veiled Blade adventurer camp ✅
- Added the Marshal Expeditions adventurer camp ✅
- The Dire trait is no longer assigned to random races that are not trolls (Fixes issue #1723 ) ✅
- Made a fix for `Call for Confederation` Decision ✅
- Characters and their vassals who flee from the scourge and travel to Kalimdor will now get the trait disinherited once Theramore is formed ✅
- Made a fix where `Commission Epic` decision was available for everyone even if they didn't have the required perk to unlock the decision ✅
- Made a fix where the Vassalization Casus Belli wasn't available after getting the `forced_vassalage_perk` ✅
- Flowerpicker culture ✅
- Non-player Tauren characters with the nickname "Brave" are more uncommon ✅
- If a non-player Tauren already has a nickname they won't get the nickname "Brave" ✅
- Religions will only get access to Great Holy Wars after the year 615 ✅
- Religions can now only target Great Holy Wars for their own Holy Sites (Fixes issue #1661 ) ✅
- Balancing Fix: Garithos won't be in the Alliance Union nor can he join it (Fixes issue #1733 ) ✅
- Chris Metzen ✅
- Characters who can breathe underwater don't drown in events ☀️ (needs more testers on this)
- Characters who can't be poisoned don't get poisoned in events ☀️ (needs more testers on this)
- Characters who can fly don't fall to their deaths ☀️ (needs more testers on this)
- Patron characters have a modifier that prevents them from autogenerating families ✅

Grekopithikos committed:
errorlog fix

Grekopithikos committed:
no duplicate murlocs

Grekopithikos committed:
Bakishroth

FelModder committed:
Portrait Fix

FelModder committed:
administrative couldn't join unions fix

Grekopithikos committed:
Man'ari eredar names

Grekopithikos committed:
slark

Grekopithikos committed:
Draenei names part 1

Grekopithikos committed:
ear_length renamed to ear_length

Grekopithikos committed:
Update 01_ethnicities_indian.txt

FelModder committed:
gotta rename the meshes too

Grekopithikos committed:
Every Draenei, Broken and Lost One name

Grekopithikos committed:
Ixthal

Grekopithikos committed:
Update 1_azerothian.txt

Grekopithikos committed:
wildhammer_shamanism

Grekopithikos committed:
Update mpo_nomads_flavour_events.txt

Grekopithikos committed:
Stratholme Service Entrance characters

edmund-h reviewed (changes requested):
@FelModder would it not be better to prevent theramore activity participants from becoming undead with a character flag that gets checked in the `become_undead_by` effects, rather than curing it after the fact? becoming undead often has other effects such as changing culture and religion that aren’t accounted for here.

hit me up on discord with your thoughts

edmund-h commented on common/culture/cultures/wc_arathi.txt:
```diff
@@ -299,4 +299,33 @@ azerothian = {
+	language = language_common
+	martial_custom = martial_custom_male_only
+	head_determination = head_determination_domain
+	traditions = { #Update this once All Under Heaven comes out
```
this can't wait until All Under Heaven comes out? how do we know this is gonna get updated?

Grekopithikos replied:
I meant that as in All Under Heaven will probably give an update to various Asian cultures so keep that in mind when it comes out

edmund-h commented on common/dynasties/ping.txt:
```diff
@@ -0,0 +1,9 @@
+﻿ping = {
+	name = "dynn_Ping"
+	culture = ping
+}
+shen = {
+	name = "dynn_Shen"
+	culture = ping
+}
+cheng = {name = "Cheng" culture = ping}
```
do any characters actually have this culture? why did we make a whole file for a culture that's not on the map and has no characters?

Grekopithikos replied:
Yes it does have characters.
[Woo Ping](https://warcraft.wiki.gg/wiki/Woo_Ping), [Wu Shen](https://warcraft.wiki.gg/wiki/Wu_Shen) and [Shen Kang Cheng](https://warcraft.wiki.gg/wiki/Shen_Kang_Cheng)

edmund-h commented on common/ethnicities/01_ethnicities_indian.txt:
```diff
@@ -64,7 +64,7 @@ indian = {
 	
 	skin_color = {
-		10 = { 0.3 0.5 0.5 0.6 }
+		10 = { 0.13 0.85 0.18 0.95 } #using african skin for now
```
these changes don't feel right to me, shouldnt we just use african then?

Grekopithikos replied:
This one is more of a bugfix, since it was like that before and then was changed to use the vanilla values which caused them to look green due to our skin colour palette.

FelModder replied:
Yes, `10 = { 0.3 0.5 0.5 0.6 }` is something i bought during compatch by mistake which caused southsea and some other cultures to have green skin.
`10 = { 0.13 0.85 0.18 0.95 } #using african skin for now` is the correct skin_color and how it was before my mistake on 1.16.2.3 compatch.

edmund-h commented on common/modifiers/wc_event_modifiers.txt:
```diff
@@ -42,17 +42,6 @@ hibernated_modifier = {
 	life_expectancy = 600
 }
 
-important_lore_character = {
-	icon = health_positive
-	
-	health = 1
-	enemy_hostile_scheme_phase_duration_add = major_scheme_phase_duration_malus_value
-	enemy_hostile_scheme_success_chance_add = -30
-	character_travel_safety = 45
-	years_of_fertility = 5
-	#flag = lower_chance_of_dying_in_combat
-}
-
```
first of all, this modifier is critical to a number of game systems. developers need to be made aware when this kind of change is made. it should not be hidden in a "hotfix".

moreover, i don't see this change accounted for in any of the game files at all, including ones modified in this very PR such as `wc_health_triggers`. revert this change ASAP. 

if you have better ideas how to handle lore characters then make another, comprehensive pull request with this change. include in the description why you want to remove it and why your replacement is better; and make sure to ping developer chat.

edmund-h replied:
oh i see it was moved to its own file. sorry this PR was so huge i couldnt see the change. resolving this issue

edmund-h reviewed (dismissed):
i have a feeling there are dozens of issues in this pull request but it's so bloated 

certain large changes such as replacing the `like_x_group` culture triggers with `like_x_heritage` should have been their own PR. i think some of the larger commits will have to be cherry-picked out before a proper code review can be done.

edmund-h commented:
i am reverting this PR to a draft because of numerous issues, some of which are game-breaking

edmund-h committed:
Revert "renamed heritage triggers"

This reverts commit b2c8c06cfda081f47fd06ebfbcbe53ea791b0a85.

edmund-h committed:
Revert "ear_length renamed to ear_length"

This reverts commit acfdb1ba934bb4a93f2bb30112441b01486cd372.

edmund-h commented on common/decisions/00_lifestyle_decisions.txt:
```diff
@@ -327,17 +327,16 @@ commission_epic_decision = {
-				# 	has_struggle_phase_parameter = unlocks_epic_commission_for_independent_rulers
-	 			# }	
+			#AND = { # Warcraft
+				#is_independent_ruler = yes
```
this line was working in vanilla, so please figure out how to restore it if possible. only rulers with no liege should be able to do this.

FelModder replied:
No? In base game, Write epic decision is available for anyone with the `writing_history_perk`.

In the base game, You can take this decision either *if you have the wriing_history_perk* OR if you are *an independent ruler involved in any struggle*. The Requirements of struggle system should not be mixed in with the basic system of having a perk.

edmund-h commented on common/scripted_triggers/wc_historical_character_triggers.txt:
```diff
@@ -1077,274 +1767,12 @@ is_easteregg22_trigger = {
 }
-
-
-#Patreons
```
please move patreon related triggers into their own file to reduce filesize. this thing is huge.

edmund-h replied:
opps you already did that. my bad. good thinking. let's give them their own category trigger as well, see below.

edmund-h commented on common/scripted_triggers/wc_historical_character_triggers.txt:
```diff
@@ -1556,6 +1987,149 @@ is_any_historical_character_trigger = {
+		is_shoopa_trigger = yes
+		is_batu_trigger = yes
+		is_chaka_trigger = yes
+		is_choa_trigger = yes
+		is_wahla_trigger = yes
+		is_lashk_trigger = yes
+		is_rauloo_trigger = yes
+		is_jol_trigger = yes
+		is_kuppaka_trigger = yes
+		is_patu_trigger = yes
+		is_akru_trigger = yes
+		is_kajosh_trigger = yes
+		is_rooka_trigger = yes
+		is_umbra_trigger = yes
+		is_ludwig_von_tortollan_trigger = yes
+		is_majo_trigger = yes
+		is_teekcha_trigger = yes
+		is_crosh_trigger = yes
+		is_kojo_trigger = yes
+		is_torka_trigger = yes
+		is_kurtu_trigger = yes
+		is_deyon_trigger = yes
+		is_nog_trigger = yes
+		is_gurtuk_trigger = yes
+		is_maedin_trigger = yes
+		is_tukuku_trigger = yes
+		is_tiang_trigger = yes
+		is_hansa_trigger = yes
+		is_shalo_trigger = yes
+		is_churka_trigger = yes
+		is_tulu_trigger = yes
+		is_kono_trigger = yes
+		is_tuka_trigger = yes
+		is_kisha_trigger = yes
+		is_soola_trigger = yes
+		is_roko_trigger = yes
+		is_toki_trigger = yes
+		is_tooka_trigger = yes
+		is_trem_trigger = yes
+		is_kee_trigger = yes
```
there must be a better solution than giving each character their own trigger.  this is lunacy.

Grekopithikos replied:
I did this for the case of tortollans in order to use those triggers for each of them to have set appearances.

edmund-h commented on common/scripted_triggers/wc_historical_character_triggers.txt:
```diff
@@ -1578,67 +2152,6 @@ is_any_historical_character_trigger = {
 		is_easteregg21_trigger = yes
 		is_easteregg22_trigger = yes
-
-		#Patreons
```
please move these into an equivalent trigger in the patreon character trigger file you made, then check that trigger separately in the same places this one is checked. then later we should have a separate PR where we evaluate whether patreons should get the same treatment if needed. we love our patreons but they are not historical characters.

Grekopithikos replied:
I did

edmund-h commented on events/dlc/mpo/mpo_nomads_flavour_events.txt:
```diff
@@ -2663,11 +2663,47 @@ nomad_events.0250 = {
 						save_scope_as = priest
 					}
 				}
+				10 = {
+					trigger = {
+						NOT = { faith = faith:wildhammer_shamanism }
+						#any_character_situation = { this = situation:the_great_steppe }
+					}
+					#modifier = {
+					#	location = { geographical_region = world_steppe_east }
+					#	add = 25
+					#}
+					create_character = {
+						location = root.location
+						template = priest_character_template
+						gender_female_chance = root_faith_clergy_gender_female_chance
+						faith = faith:wildhammer_shamanism
+						culture = culture:wildhammer
+						save_scope_as = priest
+					}
+				}
+				10 = {
+					trigger = {
+						NOT = { faith = faith:kaldorei_religion }
+						#NOT = { any_character_situation = { this = situation:game_rule_extra_nomads_sami } }
+					}
+					modifier = {
+						location = { geographical_region = world_kalimdor_north }
+						add = 25
+					}
+					create_character = {
+						location = root.location
```
are we sure this event triggers for non-nomads if there are no nomads in the game? i dont really see the point of this change otherwise.

FelModder replied:
Oh yeah it didn't. I just reflavourized it during the 1.16.2.3 compatch and forgot to make sure it triggers. Now it will.

Fixed.

Grekopithikos committed:
easter egg character triggers moved to their own file

FelModder committed:
Fixing the Broken models

Our Mod uses meshes that are named "ear_length"
Some of the meshes in the files are mistakenly named as "ear_lenght"

This has now been fixed. This is a very simple concept to understand. DO NOT  revert without asking first.

Stop Reverting just because things look big.

FelModder commented:
> @FelModder would it not be better to prevent theramore activity participants from becoming undead with a character flag that gets checked in the `become_undead_by` effects, rather than curing it after the fact? becoming undead often has other effects such as changing culture and religion that aren’t accounted for here.
> 
> hit me up on discord with your thoughts

Will do! Changing it to a character flag to stop them from converting is easier.

FelModder committed:
removing theramore vassal being_undead first

FelModder committed:
cannot become undead as a "soon to be" theramore vassal

FelModder committed:
Man of God Nomadic Event will now work with Warcraft governments

Grekopithikos committed:
Update fel_group.txt

FelModder committed:
theramore vassals and courtiers can no longer turn undead fix

TerrorBlades commented:
Not sure if I am doing this correctly, but here we go:
Two of the non generated Centaurs do not have Way of the Nomad in 583
<img width="3732" height="1016" alt="Kolkar1" src="https://github.com/user-attachments/assets/877eef97-fd48-493d-b68f-a048e9268aee" />
Edit: Not sure if they are generated or not

FelModder committed:
Cannot have children if you have `cannot_mary_historical`

This is to fix the immersion breaking and gameplay breaking issue of Jaina being soulmates with Arthas which ends up with her being pregnant almost all the time before he even turns undead.

Grekopithikos commented:
> Not sure if I am doing this correctly, but here we go: Two of the non generated Centaurs do not have Way of the Nomad in 583 <img alt="Kolkar1" width="2000" height="1016" src="https://private-user-images.githubusercontent.com/105604501/480110606-877eef97-fd48-493d-b68f-a048e9268aee.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU3MTk2MDQsIm5iZiI6MTc1NTcxOTMwNCwicGF0aCI6Ii8xMDU2MDQ1MDEvNDgwMTEwNjA2LTg3N2VlZjk3LWZkNDgtNDkzZC1iNjhmLWEwNDhlOTI2OGFlZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMFQxOTQ4MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zODExMGYxOTIyM2ExZTc1ODIwZmE0ZWVmODM2OWVmZWRmMThhM2U5MWJkN2QxNDRlM2JlMjAwMDFjYWRlMzljJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.-YpkK8K0tiP18kpLkhfpAcWGv_zGamyJVgA7WAuNNZI"> Edit: Not sure if they are generated or not

Neither of those are in the history files. They're both generated.

Grekopithikos committed:
Astalor Bloodsworn

Grekopithikos committed:
no nomadic_philosophy

Grekopithikos committed:
some gnomes and an orc

I swear these are the last ones I'm adding

Grekopithikos committed:
fixes
